docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -1,5 +1,30 @@
 # `nx-cloud` CLI
 
+## npx nx-cloud login
+
+To provision a local personal access token to access Nx Cloud features run `npx nx-cloud login`. This will open your browser at the Nx Cloud application and after signing in will generate a personal access token and save it in a configuration file locally called `nxcloud.ini`. The format of this file will look like this:
+
+```ini
+[https://cloud\.nx\.app]
+personalAccessToken=SOME_ACCESS_TOKEN
+```
+
+If you have access to multiple instances of the Nx Cloud application (e.g. self-hosted Enterprise and our managed instance), each instance will be saved to this file under its URL.
+
+Our managed instance at https://cloud.nx.app is the default. To provision a personal access token from an alternative instance of Nx Cloud pass the URL as a positional parameter, e.g.
+
+```bash
+nx-cloud login https://nx-cloud.my-domain.app
```

Comment:
Why do some of these snippets use `npx nx-cloud` and others just use `nx-cloud`

Reply:
Fair point.